app/view_objects/results_view.rb

```diff
@@ -103,6 +108,23 @@ def provider_filter?
     query_parameters["l"] == "3"
   end
 
+  def sort_by_distance?
+    query_parameters[:sortby] == DISTANCE
```

Comment:
```suggestion
    sort_by == DISTANCE
```

is `sort_by` getting used anyways?

Reply:
Yeah mate good spot. I've changed